Adding new ppl profiles

LauraUSDS opened the pull request:
Full list of new profiles in this issue: https://github.com/usds/usds/issues/5118
~Still working on images for everyone + need to upload those files.~ ~All images uploaded except Carla's.~ All pics uploaded!

LauraUSDS committed:
Create Anissa Perez profile

Still missing content for two questions

LauraUSDS committed:
Create Eunice Garcia profile

LauraUSDS committed:
Create Fatima Noor profile

still missing two questions

LauraUSDS committed:
create Neil Sharma file

still missing some questions

LauraUSDS committed:
Create Sparkle Joy profile

still missing a few questions

LauraUSDS committed:
Create carla bynum profile

LauraUSDS committed:
Create Misu Tasnim profile

LauraUSDS committed:
Create Jenn Noinaj profile

LauraUSDS committed:
Create Meg Peters profile

LauraUSDS committed:
removing unanswered questions

LauraUSDS committed:
remove unanswered questions

LauraUSDS committed:
remove unanswered questions

LauraUSDS committed:
remove unanswered questions

LauraUSDS committed:
remove unanswered questions

LauraUSDS committed:
remove unanswered questions

LauraUSDS committed:
change pic file name

bernars-usa commented on _people/0150_anissa-perez.md:
```diff
@@ -0,0 +1,51 @@
+**What will you miss most about USDS when you leave?**
+
+This is a no brainer — I will miss my colleagues most of all when I leave USDS — both USDSers and government partners alike.
+I have been blessed to have made lifelong friendships and meaningful relationships with colleagues. I mean when else in my lifetime will I have such beautifully unique colleagues like the ones I currently have?
```
```suggestion
I have been blessed to have made lifelong friendships and meaningful relationships with colleagues. I mean, when else in my lifetime will I have such beautifully unique colleagues like the ones I currently have?
```

bernars-usa commented on _people/0350_neil-sharma.md:
```diff
@@ -0,0 +1,43 @@
+**What inspired you to join USDS?**
+
+After years of struggling at startups, the excitement faded away. Most of my startups didn’t move the needle — they had users, and sometimes even made money, but were not mission critical to anyone’s lives. I wanted to feel like I was adding value, not just in a business sense but also at a personal level. I also wanted to work on something I could contribute my whole self to, not just my enginee ...
+I didn’t know much about USDS when applying; since there wasn’t a lot of content online showcasing the work, like there is today. An alumni friend described working in government as “making America better while managing a headache from face-palming every day.” I figured I had thick skin from failing at a ton of startups, and meaningfully helping people at government scale seemed like it could scr ...
```
```suggestion
I didn’t know much about USDS when applying; there wasn’t a lot of content online showcasing the work, like there is today. An alumni friend described working in government as “making America better while managing a headache from face-palming every day.” I figured I had thick skin from failing at a ton of startups, and meaningfully helping people at government scale seemed like it could scratch my itches.
```

bernars-usa commented on _people/0450_sparkle-joy.md:
```diff
@@ -0,0 +1,39 @@
+
+**What’s your background?**
+
+I’m a graduate of Stanford University with a degree in biomechanical engineering. I started my career doing research in biomechanics, but eventually transitioned to doing FDA trials and Human Subjects Research in Stroke and Traumatic Brain Injury before moving into the tech space.
```
```suggestion
I’m a graduate of Stanford University with a degree in biomechanical engineering. I started my career doing research in biomechanics, but eventually transitioned to doing FDA trials and human subjects research in stroke and traumatic brain injury before moving into the tech space.
```

bernars-usa commented on _people/0450_sparkle-joy.md:
```diff
@@ -0,0 +1,39 @@
+
+I’m a graduate of Stanford University with a degree in biomechanical engineering. I started my career doing research in biomechanics, but eventually transitioned to doing FDA trials and Human Subjects Research in Stroke and Traumatic Brain Injury before moving into the tech space.
+I’ve had the opportunity to work at several startups, but was particularly impacted by the years I spent working at Facebook.
+The biggest impetus behind my move to USDS, was the opportunity to do great work that would have a tangible impact on the lives of, literally, everyone around me.
```
```suggestion
The biggest impetus behind my move to USDS was the opportunity to do great work that would have a tangible impact on the lives of, literally, everyone around me.
```

bernars-usa commented on _people/0450_sparkle-joy.md:
```diff
@@ -0,0 +1,39 @@
+
+**What has been your biggest challenge?**
+
+Sometimes, separating from the work is really hard. Especially as I’ve spent the last few months working from home, balancing a crazy schedule of navigating COVID, working on my USDS projects, being a single mother of two, and homeschooling a Kindergartner and 4th grader! Most recently in my work life, I’ve been focused a lot on having conversations with people who are working with families who h ...
```
```suggestion
Sometimes, separating from the work is really hard. Especially as I’ve spent the last few months working from home, balancing a crazy schedule of navigating COVID, working on my USDS projects, being a single mother of two, and homeschooling a kindergartner and 4th grader! Most recently in my work life, I’ve been focused a lot on having conversations with people who are working with families who have experienced a wide range of traumatic experiences. Hearing their stories and the challenges they face can feel really heavy, particularly when the idea of developing quality solutions often feels really daunting. Despite this, I have been so encouraged by my fellow teammates who are not only incredible to work with, but also have been able to have honest, open, and transparent conversations about the work we’re doing, even when those conversations are hard.
```

bernars-usa commented on _people/0450_sparkle-joy.md:
```diff
@@ -0,0 +1,39 @@
+
+**How does your work make an impact?**
+
+The great work we do at USDS has been [on clear display during this time of pandemic and recovery](https://www.cbsnews.com/news/coronavirus-us-digital-service-technology-government/). USDS has been plugged in at agencies across the Federal government dealing with the COVID-19 response and at the end of the day, the impact that we have been able to have during this time has been astounding. Our ab ...
```
```suggestion
The great work we do at USDS has been [on clear display during this time of pandemic and recovery](https://www.cbsnews.com/news/coronavirus-us-digital-service-technology-government/). USDS has been plugged in at agencies across the Federal government dealing with the COVID-19 response and at the end of the day, the impact that we have been able to have during this time has been astounding. Our ability to navigate the massive amounts of COVID-related data has had an impact in saving lives across this country. In the aftermath of the job loss caused by extended quarantine, we’ve had teams working to ensure that Americans are able to (successfully) apply for unemployment benefits. We’ve even facilitated the government’s ability to continue working remotely so that a number of critical agencies were able to more easily transition to a fully remote workforce.
```

bernars-usa commented on _people/0450_sparkle-joy.md:
```diff
@@ -0,0 +1,39 @@
+
+The great work we do at USDS has been [on clear display during this time of pandemic and recovery](https://www.cbsnews.com/news/coronavirus-us-digital-service-technology-government/). USDS has been plugged in at agencies across the Federal government dealing with the COVID-19 response and at the end of the day, the impact that we have been able to have during this time has been astounding. Our ab ...
+Although DHS is my home agency, I’m currently working with fellow USDSers in Health and Human Services (HHS).
+Now more than ever, the American people are experiencing a national trauma that seems to get compounded day over day. USDS had already been working with agencies like SAMHSA (Substance Abuse and Mental Health Services Administration) to understand the types of challenges faced by providers and organizations all across America as they work with children who are dealing with past or present trauma.
```
```suggestion
Now more than ever, the American people are experiencing a national trauma that seems to get compounded day over day. USDS had already been working with agencies like the Substance Abuse and Mental Health Services Administration (SAMHSA) to understand the types of challenges faced by providers and organizations all across America as they work with children who are dealing with past or present trauma.
```

bernars-usa commented on _people/0550_misu-tasnim.md:
```diff
@@ -0,0 +1,41 @@
+**What’s your background?**
+
+My background is in electrical engineering, mainly telecommunications and networking. I worked for NASA at the Goddard Space Flight Center in Greenbelt, MD for three years before deciding I needed to make it my full time job to face my fear of speaking and attention.
+I stayed in the business of government contracting; learning everything from procurement, business strategy, team building, stakeholder management, and of course public speaking.
```
```suggestion
I stayed in the business of government contracting, learning everything from procurement, business strategy, team building, stakeholder management, and of course public speaking.
```

bernars-usa commented on _people/0550_misu-tasnim.md:
```diff
@@ -0,0 +1,41 @@
+
+**How does your work make an impact?**
+
+Work in government matters, USDS or not. USDS hires for empathy and passion. This allows us to make an impact by questioning the status quo and being a catalyst for action. What our team members bring to the table is a fearless energy to try and a resilience in the face of failure.
```
```suggestion
Work in government matters, USDS or not. USDS hires for empathy and passion. This allows us to make an impact by questioning the status quo and being a catalyst for action. What our team members bring to the table is a fearless energy to try, and a resilience in the face of failure.
```

bernars-usa commented on _people/0550_misu-tasnim.md:
```diff
@@ -0,0 +1,41 @@
+**How does your work make an impact?**
+
+Work in government matters, USDS or not. USDS hires for empathy and passion. This allows us to make an impact by questioning the status quo and being a catalyst for action. What our team members bring to the table is a fearless energy to try and a resilience in the face of failure.
+Often we hear “we’re the government, our systems can’t break.” Yet the government has the highest rate of failed IT projects and private technology companies continuously outpace and outperform government technology products.
```
```suggestion
Often we hear “we’re the government, our systems can’t break.” Yet, the government has the highest rate of failed IT projects. Private technology companies continuously outpace and outperform government technology products.
```

bernars-usa commented on _people/0650_eunice-garcia.md:
```diff
@@ -0,0 +1,47 @@
+
+**What inspired you to join USDS?**
+
+I first heard about USDS when I was at OPC. They had reached out to us looking for user stories from Veterans about their interactions with VA. They were very much the cool kids in the room, wearing jeans and hoodies amidst the sea of suits. What struck me was how passionate they were about their work, and that spoke to me. I am mission driven and aim to work in spaces that deliver impact, even b ...
```
```suggestion
I first heard about USDS when I was at OPC. They had reached out to us looking for user stories from Veterans about their interactions with the Department of Veterans Affairs (VA). They were very much the cool kids in the room, wearing jeans and hoodies amidst the sea of suits. What struck me was how passionate they were about their work, and that spoke to me. I am mission-driven and aim to work in spaces that deliver impact, even behind the scenes. It’s inspiring to see the work USDS has done, and continues to do, specifically in a very complicated environment in unprecedented times.
```

bernars-usa commented on _people/0650_eunice-garcia.md:
```diff
@@ -0,0 +1,47 @@
+
+**What has been your biggest challenge?**
+
+Owning my game, (a.k.a. imposter syndrome). I think I share similar sentiments, especially with many women on this one. There are days where my confidence lacks and yet you are depended on making crucial decisions for the well-being of the team.
```
```suggestion
Owning my game, (a.k.a. imposter syndrome). I think I share similar sentiments, especially with many women on this one. There are days where my confidence lacks, and yet I'm depended on to make crucial decisions for the well-being of the team.
```

bernars-usa commented on _people/0650_eunice-garcia.md:
```diff
@@ -0,0 +1,47 @@
+**What has been your biggest challenge?**
+
+Owning my game, (a.k.a. imposter syndrome). I think I share similar sentiments, especially with many women on this one. There are days where my confidence lacks and yet you are depended on making crucial decisions for the well-being of the team.
+It’s not always easy to say, “I’ve totally got this,” or “everything will be fine.” There is a lot more at play — hurdles, roadblocks, red tape, etc. and learning how to navigate that maze can be hard. I’ve learned to rely on bridges and encouraging voices in order to move forward.
```
```suggestion
It’s not always easy to say, “I’ve totally got this,” or “everything will be fine.” There is a lot more at play — hurdles, roadblocks, red tape, etc. — and learning how to navigate that maze can be hard. I’ve learned to rely on bridges and encouraging voices in order to move forward.
```

bernars-usa commented on _people/0650_eunice-garcia.md:
```diff
@@ -0,0 +1,47 @@
+
+**How does your work make an impact?**
+
+My impact is team focused. The role of being an Ops Manager is unique in the sense that a jack-of-all-trades. I see my role as an enabler, a guardian, a gatekeeper of the team.
```
```suggestion
My impact is team-focused. My role as an operations manager is unique in the sense that I'm a jack-of-all-trades. I see myself as an enabler, a guardian, a gatekeeper of the team.
```

bernars-usa commented on _people/0650_eunice-garcia.md:
```diff
@@ -0,0 +1,47 @@
+**How does your work make an impact?**
+
+My impact is team focused. The role of being an Ops Manager is unique in the sense that a jack-of-all-trades. I see my role as an enabler, a guardian, a gatekeeper of the team.
+My job is to create a healthy environment, eliminate unnecessary fires, support, guide, and facilitate. Whether it’s ensuring we have a functional work space, the right tech equipment, helping newbies find their way, or wrangling ethics clearances for an event, there are a lot of moving pieces that demand quality service on my end to ensure the team’s ability to delivery working product.
```
```suggestion
My job is to create a healthy environment, put out fires, support, guide, and facilitate. Whether it’s ensuring we have a functional work space and the right tech equipment, helping newbies find their way, or wrangling ethics clearances for an event, there are a lot of moving pieces that demand quality service on my end to ensure the team’s ability to deliver working products.
```

bernars-usa commented on _people/0650_eunice-garcia.md:
```diff
@@ -0,0 +1,47 @@
+
+**What do you want to do after USDS?**
+
+Continue to work in the civic tech space. There is definitely a need for more women of color to speak tech and government at the same time. I believe in changing the status quo, and more importantly, empowering good people with the right tools especially in government, which could bring everlasting change.
```
```suggestion
I want to continue to work in the civic tech space. There is definitely a need for more women of color to speak tech and government at the same time. I believe in changing the status quo, and more importantly, empowering good people with the right tools, especially in government, which could bring everlasting change.
```

bernars-usa commented on _people/0750_fatima-noor.md:
```diff
@@ -0,0 +1,46 @@
+
+**What’s your background?**
+
+I was born in Somalia and know firsthand what it’s like to be a refugee. The experience I had growing up in different places from Kenya to Europe, and eventually the United States, influenced my decision to take a job working with new immigrants and future Americans just arriving in the US.
```
```suggestion
I was born in Somalia and know firsthand what it’s like to be a refugee. The experiences I had growing up in Kenya, Europe, and eventually the United States, influenced my decision to take a job working with new immigrants and future Americans just arriving in the U.S.
```

bernars-usa commented on _people/0750_fatima-noor.md:
```diff
@@ -0,0 +1,46 @@
+**What’s your background?**
+
+I was born in Somalia and know firsthand what it’s like to be a refugee. The experience I had growing up in different places from Kenya to Europe, and eventually the United States, influenced my decision to take a job working with new immigrants and future Americans just arriving in the US.
+I enjoyed the work very much and wanted to scale my impact- leading me to join the Department of Homeland Security where I was assigned to the White House Domestic Policy Council to work on several federal immigration policies. I led the Taskforce on New Americans focused on the social and economic integration of our newest citizens and ran a national campaign aimed at providing assistance to loc ...
```
```suggestion
I enjoyed the work very much and wanted to scale my impact, leading me to join the Department of Homeland Security, where I was assigned to the White House Domestic Policy Council to work on several federal immigration policies. I led the Taskforce on New Americans, focused on the social and economic integration of our newest citizens, and ran a national campaign aimed at providing assistance to local governments building welcoming communities for refugees.
```

bernars-usa commented on _people/0750_fatima-noor.md:
```diff
@@ -0,0 +1,46 @@
+
+**How does your work make an impact?**
+
+Whenever we think of government services, there are regulations, policy, compliance as the backbone of these services. Not only is modernizing the technology important but understanding how to modernize the policy is also critical.
```
```suggestion
When we think of government services, regulations, policy, and compliance are the backbone of those services. Not only is modernizing the technology important, but it's also critical to understand how to modernize the policy.
```

bernars-usa commented on _people/0750_fatima-noor.md:
```diff
@@ -0,0 +1,46 @@
+**How does your work make an impact?**
+
+Whenever we think of government services, there are regulations, policy, compliance as the backbone of these services. Not only is modernizing the technology important but understanding how to modernize the policy is also critical.
+USDS really understands the need to combine these two facets of a government service — the spirit and the implementation of the service — to build usable, modern solutions that serve all Americans.
```
```suggestion
USDS really understands the need to combine these two facets of a government service — both the spirit and the implementation — to build usable, modern solutions that serve all Americans.
```

bernars-usa commented on _people/0750_fatima-noor.md:
```diff
@@ -0,0 +1,46 @@
+**What will you miss most about USDS when you leave?**
+
+The people! Without a question, the talented and awesome humans I get to call my colleagues are the best thing about USDS!
+It’s inspiring being around a group of people who are not afraid to dive into complex problems and constantly challenging themselves to think about the end user — the Veteran or the immigrant or the federal employee — relying on these services.
```
```suggestion
It’s inspiring being around a group of people who are not afraid to dive into complex problems and constantly challenge themselves to think about the end user — the Veteran or the immigrant or the federal employee — who rely on these services.
```

bernars-usa commented on _people/0750_fatima-noor.md:
```diff
@@ -0,0 +1,46 @@
+
+The people! Without a question, the talented and awesome humans I get to call my colleagues are the best thing about USDS!
+It’s inspiring being around a group of people who are not afraid to dive into complex problems and constantly challenging themselves to think about the end user — the Veteran or the immigrant or the federal employee — relying on these services.
+USDSers are the most empathetic group of people I’ve ever come across. Regardless of what problem we’re trying to solve, we always put ourselves in the shoes of the users and the people who are going to be impacted by our work.
```
```suggestion
USDSers are the most empathetic group of people I’ve ever come across. Regardless of the problem we’re trying to solve, we always put ourselves in the shoes of the users and the people who will be impacted by our work.
```

bernars-usa commented on _people/0750_fatima-noor.md:
```diff
@@ -0,0 +1,46 @@
+The people! Without a question, the talented and awesome humans I get to call my colleagues are the best thing about USDS!
+It’s inspiring being around a group of people who are not afraid to dive into complex problems and constantly challenging themselves to think about the end user — the Veteran or the immigrant or the federal employee — relying on these services.
+USDSers are the most empathetic group of people I’ve ever come across. Regardless of what problem we’re trying to solve, we always put ourselves in the shoes of the users and the people who are going to be impacted by our work.
+Not only are most people who join USDS driven by the mission, there’s a continuous challenge to ensure that our work truly addresses tangible human need. We constantly question the solutions we design and ensure they truly meet the needs of the people that we’re delivering for.
```
```suggestion
Not only are most people who join USDS driven by the mission, there’s a continuous challenge to ensure that our work truly addresses tangible human need. We constantly question the solutions we design and ensure they truly meet the needs of the people that we’re serving.
```

bernars-usa commented on _people/1150_meg-peters.md:
```diff
@@ -0,0 +1,45 @@
+
+**What inspired you to join USDS?**
+
+I joined USDS to help solve big, messy problems that make it hard for people to apply for and get government benefits. I wanted to work on teams that designed _with_ the people who would use what we designed. At USDS, we “design with users, not for them,” and we bake this process into everything we build. To design a thing well, you must understand why and how people use the thing. Sometimes you  ...
```
```suggestion
I joined USDS to help solve big, messy problems that make it hard for people to apply for and get government benefits. I wanted to work on teams that designed _with_ the people who would use what we designed. At USDS, we “design with users, not for them,” and we bake this process into everything we build. To design a thing well, you must understand why and how people use the thing. Sometimes you have to do a lot of digging to figure that out and translate it into a design that works.
```

bernars-usa commented on _people/1150_meg-peters.md:
```diff
@@ -0,0 +1,45 @@
+
+**What has been your biggest challenge?**
+
+Creating momentum has probably been the biggest challenge. Sometimes it seems impossible to move something forward, even one inch. Almost nothing we work on is small or simple, so I’ve learned to design my interactions with busy stakeholders with the same care I put into designing websites and applications. When asking stakeholders for feedback, I try to focus conversations on one thing at a time ...
```
```suggestion
Creating momentum has probably been the biggest challenge. Sometimes it seems impossible to move something forward, even one inch. Almost nothing we work on is small or simple, so I’ve learned to design my interactions with busy stakeholders with the same care I put into designing websites and applications. When asking stakeholders for feedback, I try to focus conversations on one thing at a time. This makes it easier to get thoughtful, targeted feedback, and it helps keep everyone on the same page. The scale and complexity we’re dealing with is daunting. Thousands of people apply for government benefits every day, often using fragile systems and technology that have been layered on top of one another for decades.
```

bernars-usa commented on _people/1150_meg-peters.md:
```diff
@@ -0,0 +1,45 @@
+
+**How does your work make an impact?**
+
+In three-and-a-half years at USDS, I’ve worked at IRS, Small Business Administration, U.S. Citizenship and Immigration Services, Social Security Administration, Department of Justice, and Department of Housing and Urban Development. At the Small Business Administration (SBA), my team worked with our agency partners and contractors to digitize the application and renewal process for programs that  ...
```
```suggestion
In three and a half years at USDS, I’ve worked at IRS, Small Business Administration, U.S. Citizenship and Immigration Services, Social Security Administration, Department of Justice, and Department of Housing and Urban Development. At the Small Business Administration (SBA), my team worked with our agency partners and contractors to digitize the application and renewal process for programs that help disadvantaged small businesses get government contracts. By February 2019, more than 48,000 small businesses had applied to SBA’s government contracting programs, and these programs supported 537,000 jobs and an estimated $90 billion in business across the federal government.
```

bernars-usa commented on _people/1150_meg-peters.md:
```diff
@@ -0,0 +1,45 @@
+
+**What do you want to do after USDS?**
+
+I want to keep making it easier for people to apply for and get government benefits. Since I love working on websites, I plan to do more of that.
```
```suggestion
I want to keep making it easier for people to apply for and receive government benefits. Since I love working on websites, I plan to do more of that.
```

bernars-usa commented on _people/1350_carla-bynum.md:
```diff
@@ -0,0 +1,37 @@
+
+**What’s your background?**
+
+Prior to joining the USDS at DHS team, I provided administrative support for various offices within the General Services Administration. Nine years ago, I started out supporting the GSA American Recovery Act office where I worked in the Limited Scopes office. I spent a bulk of my time with GSA supporting their Field Office and Opportunities Center at St. Elizabeths campus where we provided workfo ...
```
```suggestion
Prior to joining the USDS team at the Department of Homeland Security (DHS), I provided administrative support for various offices within the General Services Administration (GSA). Nine years ago, I started out supporting the GSA American Recovery Act office, where I worked in the Limited Scopes office, then spent the bulk of my time supporting GSA's Field Office and Opportunities Center campus at St. Elizabeths, where we provided workforce development opportunities to residents in Ward 8. I then moved to the GSA maintenance office at the Nebraska Avenue Complex.
```

bernars-usa commented on _people/1350_carla-bynum.md:
```diff
@@ -0,0 +1,37 @@
+
+**What inspired you to join USDS?**
+
+Community-service has always been a passion of mine and I was heavily involved in community service and outreach efforts at my previous job. I was attracted to USDS because of the focus on people-centered development. I love that USDS is intentional about selecting projects and is dedicated to helping underrepresented communities, such as veterans and immigrants.
```
```suggestion
Community service has always been a passion of mine, and I was heavily involved in community service and outreach efforts at my previous job. I was attracted to USDS because of the focus on people-centered development. I love that USDS is intentional about selecting projects and is dedicated to helping underrepresented communities, such as Veterans and immigrants.
```

bernars-usa commented on _people/1350_carla-bynum.md:
```diff
@@ -0,0 +1,37 @@
+
+**What will you miss most about USDS when you leave?**
+
+I will miss the family vibe USDS has. Everyone that I’ve interacted with has been super friendly and work to actively counteract stereotypes we often encounter around public servants.
```
```suggestion
I will miss the family vibe USDS has. Everyone that I’ve interacted with has been super friendly, and works to actively counteract stereotypes we often encounter around public servants.
```

bernars-usa commented on _people/1550_jenn-noinaj.md:
```diff
@@ -0,0 +1,45 @@
+
+**What inspired you to join USDS?**
+
+When I was living in Austin, I was volunteering for the local Code for America brigade and doing a lot of community organizing for my neighborhood on local issues. I believe in government services that are people-centered, are inclusive, and that work for the people who need to use them. When I heard about USDS, specifically about healthcare.gov, the efforts being made to digitize the immigration ...
```
```suggestion
When I was living in Austin, I was volunteering for the local Code for America brigade and doing a lot of community organizing for my neighborhood on local issues. I believe in government services that are people-centered, are inclusive, and that work for the people who need to use them. When I heard about USDS, specifically about healthcare.gov, the efforts being made to digitize the immigration process, and the work for Veterans, I jumped at the chance to join and make a difference at the federal level.
```

bernars-usa commented on _people/1550_jenn-noinaj.md:
```diff
@@ -0,0 +1,45 @@
+
+**What has been your biggest challenge?**
+
+My biggest challenge has been balancing the solutions that we propose with the organization’s appetite and willingness to embrace change. Even though I’m eager to roll out new initiatives to solve for all the things, I’ve learned to take a step back and determine what solutions make sense for the existing culture. Sometimes, that means that we have to do things incrementally. It’s more important  ...
```
```suggestion
My biggest challenge has been balancing the solutions that we propose with the organization’s appetite and willingness to embrace change. Even though I’m eager to roll out new initiatives to solve for all the things, I’ve learned to take a step back and determine what solutions make sense for the existing culture. Sometimes, that means that we have to do things incrementally. It’s more important that our work is sustainable, so I work with our government partners to understand where they’re coming from and to find the right path forward, together.
```

bernars-usa commented on _people/1550_jenn-noinaj.md:
```diff
@@ -0,0 +1,45 @@
+
+**How does your work make an impact?**
+
+I’ve been fortunate to work on a lot of different projects at USDS, and I think USDS’ value is in delivery and building capacity. We’re able to work on projects to take an idea to implementation, and guide teams on best practices for how to continue this momentum even after we’re gone. In terms of my work, this can mean firefighters get funding for equipment faster or helping federal agencies fin ...
```
```suggestion
I’ve been fortunate to work on a lot of different projects at USDS, and I think USDS’ value is in delivery and building capacity. We’re able to work on projects to take an idea to implementation, and guide teams on best practices for how to continue this momentum even after we’re gone. In terms of my work, this can mean firefighters get funding for equipment faster, or federal agencies can find and hire qualified talent.
```

bernars-usa reviewed (commented):
Ok! This is a SUPER light edit for punctuation, grammar, capitalization, etc. Nothing heavy-handed at all. Let me know if you have any questions!

LauraUSDS committed:
adding (most) profile pics

LauraUSDS committed:
Change Marvo to Product CoP

LauraUSDS committed:
Changing Vivian CoP

making CoP names consistent, per CoP listing in Tools. Edited from "strategy" to "strategy & operations"

LauraUSDS committed:
Update Elaine CoP

Making CoP names consistent across profiles. Removed specific title and kept 'front office'

lisac-usds commented on _people/0550_misu-tasnim.md:
```diff
@@ -0,0 +1,41 @@
+cop: Product
+city_of_origin: Gaithersburg, MD
+previously_at: Product Manager at Nuna
+image_name: misu-tasnim.jpg
```
```suggestion
image_name: misu-tasnim.png
```

LauraUSDS committed:
upload Carla's pic

LauraUSDS committed:
fix filename for Misu pic

LauraUSDS committed:
Update _people/0150_anissa-perez.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0350_neil-sharma.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0450_sparkle-joy.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0450_sparkle-joy.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/1350_carla-bynum.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/1550_jenn-noinaj.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/1550_jenn-noinaj.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0450_sparkle-joy.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0450_sparkle-joy.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/1350_carla-bynum.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/1550_jenn-noinaj.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0450_sparkle-joy.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0550_misu-tasnim.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0550_misu-tasnim.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0550_misu-tasnim.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0650_eunice-garcia.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0650_eunice-garcia.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0650_eunice-garcia.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0650_eunice-garcia.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0650_eunice-garcia.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0650_eunice-garcia.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0750_fatima-noor.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0750_fatima-noor.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0750_fatima-noor.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0750_fatima-noor.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0750_fatima-noor.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0750_fatima-noor.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/0750_fatima-noor.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/1150_meg-peters.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/1150_meg-peters.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/1150_meg-peters.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/1150_meg-peters.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS committed:
Update _people/1350_carla-bynum.md

Co-authored-by: Sheri Bernard Trivedi <40181132+bernars-usa@users.noreply.github.com>

LauraUSDS commented:
@lisac-usds @bernars-usa  ok! I think this is all set to push to staging (so I can double-check for any weirdness). any reason why i shouldn't go ahead and merge this pull request?

lisac-usds commented:
i think you should go for it, @LauraUSDS !

drew-usds commented:
@LauraUSDS you can merge to staging ... but ... It might get blocked from going _live_ live on account of the currently open security alert issue #559
